seq-mainnet-1/README.md

```diff
@@ -36,392 +27,19 @@ Alternate public nodes:
 - **Public RPC**: https://fuel-rpc.polkachu.com
 - **Public RPC**: https://rpc-fuel.kintsugi-nodes.com
 
-Indexer:
+For a list of historical Sequencer upgrades, refer to the [upgrades doc](./UPGRADES.md).
 
-- `/seq/withdrawals?address=ADDRESS` ([example](https://indexer-fuel-seq.simplystaking.xyz/seq/withdrawals?address=0x0165878A594ca255338adfa4d48449f69242Eb8F))
-- `/seq/withdrawals?address=ADDRESS&pending=true` ([example](https://indexer-fuel-seq.simplystaking.xyz/seq/withdrawals?address=0x0165878A594ca255338adfa4d48449f69242Eb8F&pending=true))
-- `/seq/supply-deltas` ([example](https://indexer-fuel-seq.simplystaking.xyz/seq/supply-deltas))
-- `/seq/supply-deltas?pending=true` ([example](https://indexer-fuel-seq.simplystaking.xyz/seq/supply-deltas?pending=true))
-- `/seq/supply-deltas?pending=true&zeros=false` ([example](https://indexer-fuel-seq.simplystaking.xyz/seq/supply-deltas?pending=true&zeros=false))
-- `/seq/proof?nonce=NONCE` ([example](https://indexer-fuel-seq.simplystaking.xyz/seq/proof?nonce=1))
+## Run the Sequencer
 
-Ethereum contracts ([ref](https://github.com/FuelLabs/fuel-rollup/tree/main/networks)):
+> **NOTE**: the guides assume some knowledge around running Cosmos SDK based nodes. It is highly recommended to familiarise yourself with this type of node operation before running a Fuel Sequencer node.
 
-- [0x675B68AA4d9c2d3BB3F0397048e62E6B7192079c](https://etherscan.io/address/0x675B68AA4d9c2d3BB3F0397048e62E6B7192079c): TokenV2
-- [0x56EBDaE96d179549f279ea0cFEa3b3432B8Cd2bC](https://etherscan.io/address/0x56EBDaE96d179549f279ea0cFEa3b3432B8Cd2bC): TokenV1
-- [0xd57d30D06969E5A98516e9f8D009C6F39EC169eE](https://etherscan.io/address/0xd57d30D06969E5A98516e9f8D009C6F39EC169eE): Vault
-- [0xBa0e6bF94580D49B5Aaaa54279198D424B23eCC3](https://etherscan.io/address/0xBa0e6bF94580D49B5Aaaa54279198D424B23eCC3): SequencerProxy (sidecar listens to this)
-- [0xca0c6B264f0F9958Ec186eb2EAa208966187D866](https://etherscan.io/address/0xca0c6B264f0F9958Ec186eb2EAa208966187D866): SequencerInterface (users use this to deposit etc.)
-- [0x50Ed39b58f66338B84E67e6Ff9a2bf00725EdFE8](https://etherscan.io/address/0x50Ed39b58f66338B84E67e6Ff9a2bf00725EdFE8): TokenMigrator
-- [0x481aeEB9bdFe08f050d22F0b352356691c4B0b59](https://etherscan.io/address/0x481aeEB9bdFe08f050d22F0b352356691c4B0b59): FuelStreamX (bridge commitments and withdrawals/supplydelta)
+There are two main options for running the Sequencer:
 
-Consensus addresses of genesis validators:
+- Run a Sequencer Full Node (non-Validator).
```

Comment:
Running a Full node requires running the sidecar if the full node is going to be used for providing data, otherwise throughput would be compromised whenever the full node is validating blocks.